### ISSUE TITLE
Add unit tests

### ISSUE THREAD

**Issue by florisvdh** [r-spatial/rgeopackage#5]
Since the package may be going to expand substantially (#4), it will be wise to add unit tests as well as automated package checks on different platforms in GH Actions, in order to maintain and guarantee functionality in the future.

Do you agree @elipousson ?

**Comment by elipousson** [comment]
Yep! I just added a test with testthat on my fork and added an empty GeoPackage file for testing on. I already use codecov.io on several other packages if you're comfortable with using that here.

**Comment by florisvdh** [comment]
Cool! It would be great if you could add the covr + codecov workflow indeed.
